x-pack/plugins/transform/public/app/sections/transform_management/components/stats_bar/stats_bar.tsx

```diff
@@ -29,7 +31,9 @@ interface StatsBarProps {
 }
 
 export const StatsBar: FC<StatsBarProps> = ({ stats, dataTestSub }) => {
-  const statsList = Object.keys(stats).map((k) => stats[k as StatsKey]);
+  const statsList = Object.keys(stats)
+    .map((k) => stats[k as StatsKey])
+    .filter<StatsBarStat>(isDefined);
```

Comment:
is this filter definitely needed?
if `nodes` hasn't been added to the `stats` it won't be there and so won't need filtering out.